imports/api/generate-data.app-test.js

```diff
@@ -6,10 +6,7 @@
 // ensuring the method is always available
 
 import { Meteor } from 'meteor/meteor'
-import { Factory } from 'meteor/dburles:factory'
```

Comment:
If some of these packages are not used in whole project, we should also uninstall them (remove them from packages. json, or wherever they live - either meteor uninstall or meteor npm uninstall)

Reply:
Okay, that's something to look into as well. We can try to figure out which packages to remove on monday. Standard.js (the linter) basically pointed out all the locations where imports were necessary, so I had removed.